api/v1alpha1/openapi.yaml

```diff
@@ -3958,7 +3958,14 @@ components:
           type: string
           description: An opaque string that identifies the server's internal version of an object.
       description: ObjectMeta is metadata that all persisted resources must have, which includes all objects users must create.
-
+    DeviceMetadata:
+      allOf:
+        - $ref: '#/components/schemas/ObjectMeta'
+        - type: object
+          properties:
+            alias:
```

Comment:
do we need to update any docs on this?

Reply:
Yes, we do; I'll update you once they are ready for review.
They will be added to this PR.